src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -285,7 +285,7 @@ private BoundExpression FixTupleLiteral(ArrayBuilder<DeconstructionVariable> che
                 Conversion nestedConversion;
                 if (variable.HasNestedVariables)
                 {
-                    var elementSyntax = syntax.Kind() == SyntaxKind.TupleExpression ? ((TupleExpressionSyntax)syntax).Arguments[i] : syntax;
+                    _ = syntax.Kind() == SyntaxKind.TupleExpression ? ((TupleExpressionSyntax)syntax).Arguments[i] : syntax;
```

Comment:
this seems to indicate a bug in the original compiler code, or just pointless computation.  so i do like this.